### PR TITLE
Staging area fixes (#366, #382, #384)

### LINKED ISSUE #366: Staging area overlaps suggestions area
screen shot 2016-10-18 at 14 12 20

### LINKED ISSUE #382: Ability to "block" chat UI completely
screen shot 2016-10-21 at 15 21 51

### LINKED ISSUE #384: Browse input in wrong place
ww

### PR DESCRIPTION
**Pull request by alwx**
fixes #366
fixes #382 
fixes #384

### CODE REVIEW

**Comment by rasom** [comment]
There is no border between staging area and message input in design

screen shot 2016-10-26 at 09 34 22

**Comment by rasom** [comment]
Staging area overlaps request header:

req

**Comment by rasom** [comment]
When the third command is added to the staging area you need to scroll down to find it, and this gives an impression that nothing happened. Maybe it would be better to scroll view down programmatically and show the last command:

lst

I'm not sure that the distance between staged commands is specified in the desing, but it feels like it is too big at the moment:

screen shot 2016-10-26 at 10 06 13

**Comment by jarradh** [comment]
yeah thats huge, ideally it would be much, much smaller, we dont have any specifications for this, but i imagine a 1/5th of this would look better

**Comment by rasom** [comment]
There is another problem when the command is removed from the staging area

android:

123a

ios:

123i

**Comment by alwx** [comment]
Done

File: android/app/build.gradle [hunk 1]
```diff
@@ -93,14 +93,6 @@ android {
             abiFilters "armeabi-v7a", "x86"
         }
     }
-    signingConfigs {
```
Comment (rasom) [inline android/app/build.gradle]: Why did you removed these lines?

**Comment by rasom** [comment]
bugs related to #366, #382 are fixed, looks fine now

**Comment by alwx** [comment]
This PR now includes 3 bugfixes, and it is not in progress anymore. Please, check and verify.

**Pull request merged**